backend/app.py

```
@@ -286,5 +390,245 @@ def evaluate_model():
+def get_triage_summary(case_id):
+    """Return the triage summary for a case, or null if not yet scored."""
+    from backend.seed_data import TRIAGE_BY_CASE
+    summary = TRIAGE_BY_CASE.get(case_id)
+    if summary is None:
+        return jsonify(None)
+    return jsonify(summary)
+
+
 if __name__ == "__main__":
-    app.run(host="0.0.0.0", port=5000, debug=True)
+    app.run(host="0.0.0.0", port=5001, debug=True)
```

Ruff (0.16.2)

[error] Possible binding to all interfaces

(S104)